src/broker/shutdown.c

```diff
@@ -63,6 +63,7 @@ void shutdown_destroy (shutdown_t *s)
     if (s) {
         if (s->shutdown)
             flux_msg_handler_destroy (s->shutdown);
+        shutdown_disarm (s);
```

Comment:
Perhaps a good change, but unrelated to the logical topic of this commit.

Reply:
Yep.